lib/ClusterShell/Defaults.py

```diff
@@ -57,7 +57,12 @@ def _load_workerclass(workername):
 
     The module is loaded if not done yet.
     """
-    modname = "ClusterShell.Worker.%s" % workername.capitalize()
+
+    # Allow short module names for built-in workers
+    if workername in [ 'exec', 'pdsh', 'rsh', 'ssh' ]:
+        modname = "ClusterShell.Worker.%s" % workername.capitalize()
```

Comment:
first try this method, if it fails, try importing 'workername', and let this exception being raised if that fails too.

Reply:
Sounds fine to me.  I broke the actual loading into a separate function to reduce repeating the code, hope that's OK